include/swift/AST/KnownIdentifiers.def

```diff
@@ -118,7 +118,6 @@ IDENTIFIER(withArguments)
 IDENTIFIER(withKeywordArguments)
 
 // SWIFT_ENABLE_TENSORFLOW
```

Comment:
I guess this comment can be removed as well?

Reply:
It should be kept, because the other identifiers below are also `tensorflow`-branch only.